blog/2022-06-30-release/index.md

```diff
@@ -0,0 +1,176 @@
+---
+slug: release-v0.5.0
+title: "Koordinator v0.5: Now With Node Resource Topology And More"
+authors: [jason]
+tags: [release]
+---
+
+In addition to the usual updates to supporting utilities, Koordinator v0.5 adds a couple of new useful features we think
+you'll like.
+
+## Install or Upgrade to Koordinator v0.5.0
+
+### Install with helms
```

Comment:
1. kubelet readonly port adaption
2. more parameters and service in runtime proxy protocol #231 #243 #239 #252
3. proxy restart panic fix #246
4. cpu usae precision #221
5. cpu fine-grained scheduling
6. standalone running mode for koordlet-runtime-hooks
7. using cache to improve slo-controller configmap management

Reply:
8. qos manager proposal
9. resource reservation proposal